### ISSUE TITLE
Stuck in the Menu Bar mode.

### ISSUE THREAD

**Issue by nextofsearch** [kartik-venugopal/aural-player#112]
Hi,

I am using the Menu Bar Mode and can't switch to other window modes even though I select them using the radio buttons on the popup window. Please take a look at the attached image.

<img width="382" alt="Screenshot 2024-11-12 at 5 05 18 PM" src="https://github.com/user-attachments/assets/d92f8147-a7e4-4c08-b54e-e5f5c64f3e36">

I was using version 3.x initially and it crashed often when using the menu bar mode so I installed version 4.x. That's when the problem started. I also installed the latest one (preview 13). Nothing helps.

**Comment by kartik-venugopal** [comment]
Hi, thanks for the issue report. I will look into fixing it ASAP.

**Comment by kartik-venugopal** [comment]
I took a quick look, and was unable to reproduce the issue. I noticed that in your screenshot, the radio buttons don't appear properly, and the OK button also doesn't appear for you (see my screenshot).

I need more info from you - what's your hardware (Intel / Apple Silicon) and macOS version ?

Also, can you send me the log file: ~/Music/aural/aural.log ? Thanks.

**Comment by nextofsearch** [comment]
Thank you so much for the quick response.

Apple M1 Max
Sonoma 14.7

I attached the log file.

////

Just before sending this reply, I opened Aural to check out the differences
in UI as you mentioned, and then I found out there is the 'OK' button back!
I don't know what I did after posting this issue on GitHub but nothing
particularly related to Aural player. Anyway, it's working now. Very strange
.
[image: Screenshot 2024-11-12 at 10.04.39 PM.png]


2024년 11월 12일 (화) 오후 8:03, Kartik Venugopal ***@***.***>님이 작성:

**Comment by nextofsearch** [comment]
[aural.log](https://github.com/user-attachments/files/17717028/aural.log)
Here's the log file.

**Comment by kartik-venugopal** [comment]
Thanks for the info. I upgraded to the same macOS version as you - 14.7.1 and I am still unable to reproduce the issue.

To be honest, I think it could just be a macOS UI bug ... sometimes I have had sporadic UI issues in Aural that are not due to the app but caused by the OS.

I'll keep this issue open for now ... let's keep an eye on the Menu Bar mode and see if the issue comes back. If and when it happens again, try to note down any patterns of what might have caused the issue (anything you did with the app just before it happened).

In the meantime, I will also experiment with changing the presentation modes panel in Menu Bar mode (that box that lets you change the mode) ... maybe regular push buttons instead of radio buttons, and see if that fixes the issue.

**Comment by kartik-venugopal** [comment]
I have redesigned the UI panel that lets you change the presentation mode. Instead of 4 radio buttons + the OK button, there are now 4 regular push buttons that will immediately switch the mode without needing to click OK. See the attached image.

Released in preview14. Try it out and let me know how it works for you.

Will keep this issue open for a while, till we are satisfied that the new UI works fine.

![Screenshot 2024-11-12 at 11 39 20 PM](https://github.com/user-attachments/assets/f01b57c3-54f0-4ea5-af24-19ad5dee3579)
